Makefile

```diff
@@ -2,36 +2,98 @@
 all: build
 .PHONY: all
 
-IMAGE_REGISTRY?=quay.io
-
 # Include the library makefile
 include $(addprefix ./vendor/github.com/openshift/build-machinery-go/make/, \
 	golang.mk \
 	targets/openshift/deps.mk \
 	targets/openshift/images.mk \
 	targets/openshift/bindata.mk \
+	lib/tmp.mk\
 )
 
+IMAGE_REGISTRY?=quay.io/open-cluster-management
+OPERATOR_SDK?=$(PERMANENT_TMP_GOPATH)/bin/operator-sdk
+OPERATOR_SDK_VERSION?=v0.17.0
+operatorsdk_gen_dir:=$(dir $(OPERATOR_SDK))
+KUBECTL?=kubectl
+KUBECONFIG ?= ./.kubeconfig
+OLM_NAMESPACE?=olm
+
+OPERATOR_SDK_ARCHOS:=x86_64-linux-gnu
+ifeq ($(GOHOSTOS),darwin)
+	ifeq ($(GOHOSTARCH),amd64)
+		OPERATOR_SDK_ARCHOS:=x86_64-apple-darwin
+	endif
+endif
+
 $(call add-bindata,hub,./manifests/hub/...,bindata,bindata,./pkg/operators/hub/bindata/bindata.go)
 $(call add-bindata,spoke,./manifests/spoke/...,bindata,bindata,./pkg/operators/spoke/bindata/bindata.go)
 
 copy-crd:
 	bash -x hack/copy-crds.sh
 
-update-all: copy-crd update-bindata-hub update-bindata-spoke
+update-all: copy-crd update-bindata-hub update-bindata-spoke update-csv
 
 verify-crds:
 	bash -x hack/verify-crds.sh
 
 verify: verify-crds
 
+update-csv: ensure-operator-sdk
+	$(OPERATOR_SDK) generate csv --crd-dir=deploy/nucleus-hub/crds --deploy-dir=deploy/nucleus-hub --output-dir=deploy/nucleus-hub/olm-catalog/nucleus-hub --operator-name=nucleus-hub --csv-version=0.1.0
+	$(OPERATOR_SDK) generate csv --crd-dir=deploy/nucleus-spoke/crds --deploy-dir=deploy/nucleus-spoke --output-dir=deploy/nucleus-spoke/olm-catalog/nucleus-spoke --operator-name=nucleus-spoke --csv-version=0.1.0
+
+deploy: install-olm deploy-hub deploy-spoke
+
+clean-deploy: clean-spoke clean-hub
+
+install-olm: ensure-operator-sdk
+	$(OPERATOR_SDK) olm status --olm-namespace $(OLM_NAMESPACE) ; if [ $$? -ne 0 ] ; then $(OPERATOR_SDK) olm install --version 0.14.1 ; fi
```

Comment:
They restored the sha that was preventing 0.15+ from working correctly with this command, so we can probably just install the latest version now.